yarn-project/aztec-cli/src/client.ts

```diff
@@ -56,6 +56,9 @@ class VersionMismatchError extends Error {}
 export async function checkServerVersion(rpc: AztecRPC, expectedVersionRange: string, logger?: DebugLogger) {
   const serverName = 'Aztec Sandbox';
   const { client } = await rpc.getNodeInfo();
+  if (!client) {
+    throw new VersionMismatchError(`Couldn't determine ${serverName} version. You may run into issues.`);
```

Comment:
Do you know why this would happen and perhaps the "You may run  into issues" part is superfluous?

Reply:
So I only added this because somebody ran into this when they hadn't updated their Docker image and the `getNodeInfo` function didn't exist. It should never happen but if it does, it's a better error message than `Cannot read properties of undefined (reading 'split')`

Reply:
Perhaps "No response received from Sandbox" or something?

Reply:
Ah yeah thats a good point re it being a better message